crates/variables/src/provider.rs

```diff
@@ -1,2 +1,3 @@
+pub mod azkv;
```

Comment:
`azure_kv` or `azure_key_vault` might be a more self-explanatory name.

Reply:
That's fair. I renamed the mod as part of 0a3e792